packages/manager-react-components/src/components/datagrid/datagrid.component.tsx

```diff
@@ -68,11 +97,17 @@ export interface DatagridProps<T> {
   /** setSorting?: OnChangeFn<SortingState>; */
   /** label displayed if there is no item in the datagrid */
   noResultLabel?: string;
+  /** List of filters and handlers to add, remove */
+  filters?: FilterProps;
+  /** Enables columns filters */
+  isFilterable?: boolean;
 }
 
 export const Datagrid = <T,>({
   columns,
   items,
+  filters,
+  isFilterable = true,
```

Comment:
**question:** Why do we add this props? We only accept it from columnDef no?

Reply:
Right, we define it together but although it detect column filterable in column definition it can disable or not the filter feature.
But we can remove it